### PR TITLE
Add account_invoice_uom

### PR DESCRIPTION
**Pull request by eLBati**
The module displays the internal UoM and quantity on the invoice lines,
retrieving them from the linked sale order(s), or from the picking lines
(when the invoice is based on delivery).

### CODE REVIEW

**Comment by tafaRU** [comment]
@eLBati, you can find some improvements at https://github.com/eLBati/account-invoicing/pull/3

**Comment by eLBati** [comment]
@tafaRU thanks

**Comment by tafaRU** [comment]
@eLBati the yml test failed, could you please investigate about it?

The rest LGTM.

**Comment by eLBati** [comment]
:white_check_mark:

**Comment by tafaRU** [comment]
:+1:

**Comment by archetipo** [comment]
:+1:

File: account_invoice_uom/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "Unit of measure for invoices",
+    "version": "1.0",
+    "author": "Agile Business Group",
```
Comment (tafaRU) [inline account_invoice_uom/__openerp__.py]: Please add OCA as author.

**Pull request merged**